samples/lightdb_led/README.rst

```diff
@@ -213,4 +219,5 @@ as:
 .. _Networking with QEMU: https://docs.zephyrproject.org/latest/guides/networking/qemu_setup.html#networking-with-qemu
 .. _ESP32: https://docs.zephyrproject.org/latest/boards/xtensa/esp32/doc/index.html
 .. _AT Binary Lists: https://docs.espressif.com/projects/esp-at/en/latest/AT_Binary_Lists/index.html
+.. _flashing overview: https://docs.golioth.io/hardware/nrf91/quickstart/flash-firmware/flash-firmware
```

Comment:
I think that there should be a one way direction of dependencies, not back and forth. In this specific case I think that `https://docs.golioth.io` should rather reference what is documented in SDK repo, not the other way.